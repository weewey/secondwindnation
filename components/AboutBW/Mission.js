import React from 'react';
import Typography from 'material-ui/Typography';
import { withStyles } from 'material-ui/styles';
import PropTypes from 'prop-types';

const styleSheet = theme => ({
  root: {
    width: '100%',
    height: '60vh',
    maxHeight: 400,
    textAlign: 'center',
    // backgroundColor: 'orange',
    justifyContent: 'center',
    display: 'flex',
    alignItems: 'center',
  },
  para: {
    width: '40em',
    padding: 20,
    margin: 'auto',
  },
  text: {
    marginTop: 16,
    fontSize: '1.5em',
    fontWeight: 100,
  },
  title: {
    // fontWeight: 500,
    color: 'black',
  },
});

function Mission(props) {
  const classes = props.classes;
  return (
    <div className={classes.root}>
      <div className={classes.para}>
        <Typography type="display3" className={classes.title}>MISSION</Typography>
        <Typography type="subheading" className={classes.text}>To provide resources for swim bike run enthusiasts to achieve
breakthroughs. To build a community that helps one another
find their Second Wind.</Typography>
      </div>
    </div>
  );
}


Mission.propTypes = {
  classes: PropTypes.object.isRequired,
};

export default withStyles(styleSheet)(Mission);
