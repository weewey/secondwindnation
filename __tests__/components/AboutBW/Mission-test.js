import React from 'react';
import renderer from 'react-test-renderer';

import Mission from '../../../components/AboutBW/Mission';

describe('Mission component', () => {
  it('should render', () => {
    const mission = renderer.create(<Mission />,);
    const tree = mission.toJSON();
    expect(tree).toMatchSnapshot();
  },
);
});
