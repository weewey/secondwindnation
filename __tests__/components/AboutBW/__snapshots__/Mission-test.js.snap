// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission component should render 1`] = `
<div
  className="Mission-root-1"
>
  <div
    className="Mission-para-2"
  >
    <h1
      className="MuiTypography-root-5 MuiTypography-display3-7 Mission-title-4"
    >
      MISSION
    </h1>
    <h3
      className="MuiTypography-root-5 MuiTypography-subheading-12 Mission-text-3"
    >
      To provide resources for swim bike run enthusiasts to achieve breakthroughs. To build a community that helps one another find their Second Wind.
    </h3>
  </div>
</div>
`;
